CMakeLists.txt

```diff
@@ -263,13 +263,3 @@ endif()
 if(TARGET doc)
   file(COPY ${CMAKE_SOURCE_DIR}/tutorials/files/ DESTINATION ${CMAKE_BINARY_DIR}/doxygen/html/files/)
 endif()
-
-# TICKTOCK MASTER HEADER (to bypass GZ_DESIGNATION)
-# TODO(CH3): Deprecated. Remove on tock
-configure_file(
```

Comment:
No idea what this was actually doing. AFAIK, `GZ_DESIGNATION` will be `sim`, so this is a noop?

Reply:
Same here.  It said remove in 8, so I did.  Maybe @methylDragon has an idea?